targets/mbed/source/jerry_extapi.cpp

```diff
@@ -27,12 +27,12 @@
 
 //-----------------------------------------------------------------------------
 
-#define __UNSED__ __attribute__((unused))
+#define __UNUSED__ __attribute__((unused))
```

Comment:
I've just realized that this file is from the mbed target. This PR should not change other targets but ESP alone. Even if the mbed target contains questionable parts. Fixing those should be left for another PR.